### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ryanjorgeac**

### CODE REVIEW

File: cursor.py [hunk 1]
```diff
@@ -20,7 +23,8 @@ def select(self,columns,fromTable,where):
             for line in archive:
                 line = line[:-1]
                 logindoc, passworddoc, saldo = line.split(":")
-                if login == logindoc:
-                    listRegister.append({"login":logindoc,"password":passworddoc, "saldo":saldo})
-                    break
+                register = {"login":logindoc,"password":passworddoc, "saldo":saldo}
```
Comment (petrolifero) [inline cursor.py:26]: why select know the column's names if is open to use any table, ```        with open(f"{fromTable}.txt") as archive:``` ???

File: test_contas.py [hunk 2]
```diff
@@ -40,10 +45,10 @@ def test_authentication_just_or_nothing():
 
 def test_authentication_on_second_account():
     archive = io.StringIO("ryan:35ca097a8dab8ede4d632f41c909a3516a259e3b954f55b081f76e627d8a85cb81e91504a8fafc25e3a9074657550f6649029dca4b8c4253a67254b57b04131c:17\npedro:c70b5dd9ebfb6f51d09d4132b7170c9d20750a7852f00680f65658f0310e810056e6763c34c9a00b0e940076f54495c169fc2302cceb312039271c43469507dc:1400\n")
-    c = contas.contas(archive).authentication("pedro","abc123")
+    c = contas.contas(cursor.linearCursor()).authentication("pedro","abc123")
     assert c
 
 def test_authentication_on_second_account_with_real_archive():
```
Comment (petrolifero) [inline test_contas.py:51]: this test fails

File: cursor.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
+import condition
+
+
 class linearCursor():
 
     def insert(self,values,into):
```
Comment (petrolifero) [inline cursor.py:6]: insert is supposed to be general, but he knows the collumns name?

**Review by petrolifero — CHANGES_REQUESTED** [review]

File: condition.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    def __init__(self, string):
+        self.string = string
+
+    def __str__(self):
+        return f"'{self.string}'"
+
+    def proj(self,dicio):
+        return self.string
+
+class columnname(projection):
```
[the remark this answers is not in the document]
Comment (ryanjorgeac) [reply condition.py]: Done.

**Review by petrolifero — DISMISSED** [review]

**Review by petrolifero — APPROVED** [review]

**Pull request merged**